src/Controls/src/Core/VisualElement.cs

```diff
@@ -801,7 +799,14 @@ public ResourceDictionary Resources
 		public bool Focus()
 		{
 			if (IsFocused)
+			{
+#if ANDROID
+				// TODO: Refactor using mappers for .NET 8
+				if (this is ITextInput && Handler?.PlatformView is not null)
+					KeyboardManager.ShowKeyboard(this.ToPlatform());
```

Comment:
I tried that initially, but changed it because of these issues

![image](https://user-images.githubusercontent.com/21988533/223188721-274c2097-6623-4d4a-b2bb-c3e46f6e976e.png)

Reply:
```C#
if (Handler is IPlatformViewHandler platformViewHandler)
     platformViewHandler.PlatformView
```

Reply:
I ended up casting instead to avoid another conditional